lisa/sut_orchestrator/baremetal/build.py

```diff
@@ -24,6 +26,28 @@ def __init__(self, runbook: BuildSchema) -> None:
     def type_schema(cls) -> Type[schema.TypedSchema]:
         return BuildSchema
 
+    @classmethod
+    def find_matched_files(
```

Comment:
Move it to a private function in this file.

Reply:
The change is done, thanks